types/src/transaction/transaction_v1/transaction_v1_hash.rs

```diff
@@ -23,13 +23,8 @@ use crate::{
     Copy, Clone, Ord, PartialOrd, Eq, PartialEq, Hash, Serialize, Deserialize, Debug, Default,
 )]
 #[cfg_attr(feature = "datasize", derive(DataSize))]
-#[cfg_attr(
-    feature = "json-schema",
-    derive(JsonSchema),
-    schemars(with = "String", description = "Hex-encoded TransactionV1 hash.")
-)]
```

Comment:
I think we could reinstate this, but remove only the `with = "String", ` part, as it's misleading.  Having the generated schema as
```json
      "allOf": [
        {
          "$ref": "#/definitions/Digest"
        }
      ]
```
is ok for this type, as `Digest` is correctly identified as a string type in the schema.